met-web/src/components/engagement/form/EngagementFormTabs/FormTabs.tsx

```diff
@@ -7,9 +7,31 @@ import { EngagementFormTabValues, ENGAGEMENT_FORM_TABS } from './constants';
 import EngagementUserManagement from './UserManagement/EngagementUserManagement';
 import EngagementAdditionalDetails from './AdditionalDetails';
 import EngagementSettings from './Settings';
+import { ActionContext } from '../ActionContext';
+import { MetTooltip } from 'components/common';
 
 const FormTabs = () => {
     const [value, setValue] = React.useState<EngagementFormTabValues>(ENGAGEMENT_FORM_TABS.CONTENT);
+    const { savedEngagement } = useContext(ActionContext);
+
+    const generateTab = (label: string, value: string, icon = null) => {
+        return (
+            <MetTab
+                style={{ pointerEvents: 'auto' }}
+                disabled={!savedEngagement.id || savedEngagement.id === 0}
```

Comment:
Is the intent here that if the ID is equal to zero then the component is disabled? If so, I think you can just do:

```suggestion
                disabled={!savedEngagement.id}
```

...since 0 is falsy in JS

Reply:
Updated as suggested, thanks.